### PR TITLE
Create Moyo-Made.json

### PR DESCRIPTION
**Pull request by Moyo-Made**
## Check List (Check all the applicable boxes) 

- [X ] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X ] I have updated the documentation accordingly.
- [ X] All new and existing tests passed.
- [X ] This PR does not contain plagiarized content.
- [X ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/Moyo-Made.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Moyo-Made",
+  "bio": "Software Developer | JavaScript | React | Open to opportunities"
```
Comment (sital002) [inline data/Moyo-Made.json:3]:
```suggestion
  "bio": "Software Developer | JavaScript | React | Open to opportunities",
```
please add a comma here

File: data/Moyo-Made.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      "url": "http://madev7.netlify.app",
+      "icon": "FaLink"
+    },
+     {
+      "group": "Socials Primary",
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/moyomadee7",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    },
```
Comment (sital002) [inline data/Moyo-Made.json:17]:
```suggestion
    }
```
The trailing comma isnot required

**Review by sital002 — CHANGES_REQUESTED** [review]
Please make the necessary changes to get your profile merged

**Review by Moyo-Made — COMMENTED** [review]
@sital002 I've made the required changes.

**Review by sital002 — APPROVED** [review]
Looks good 👍
Welcome to Eddiehub community 🎉🎉

**Comment by Moyo-Made** [comment]
> Looks good 👍
> 
> Welcome to Eddiehub community 🎉🎉

Thank you💪🏾

**Pull request merged**